src/haskell/devcontainer-feature.json

```diff
@@ -0,0 +1,53 @@
+{
+    "id": "haskell",
+    "version": "1.0.0",
+    "name": "Glasgow Haskell Compiler",
+    "documentationURL": "https://github.com/devcontainers/features/tree/main/src/haskell",
+    "description": "Installs the Glasgow Haskell Compiler",
+    "options": {
+        "version": {
+            "type": "string",
+            "proposals": [
```

Comment:
I'm not too familiar with how features work. My feeling is that this list can become outdated quickly. For HLS, it already is. Would it not be enough to choose between recommended and latest? Can the user enter manual strings regardless?